addons/udes_suggest_location/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import models
```

Comment:
nit: Name of module maybe should be in plural udes_suggest_locations, checking from description it is suggesting some locations.
Changing name maybe its tricky because of data xml etc, you can ignore it